### DISCUSSION TITLE
Mode Tabs improvements

### DISCUSSION THREAD

**Discussion by janwiesemann** [isontheline/pro.webssh.net#1085] [Ideas]
1. The selected tab of the connection mode should persist after a app restart. This would avoid an additional selection list, if you’re mostly using just SSH or SFTP.  
![IMG_6770](https://github.com/isontheline/pro.webssh.net/assets/1347201/95e6d446-734b-4e5b-bb33-8a27d6a90d92)

3. A additional indication of the selected mode should be added. This would make it a lot clearer more you where currently using. Maybe a icon or a pinned tab bar?  
![IMG_6771](https://github.com/isontheline/pro.webssh.net/assets/1347201/c22202f5-2a5c-46e7-914e-f2122c6b0552)

**Comment by isontheline** [comment]
@janwiesemann You can set your favorite startup view :
<img width="247" alt="image" src="https://github.com/isontheline/pro.webssh.net/assets/44212923/4e5e85e3-2fe8-4f20-82f9-c1b842ec1852">
<img width="247" alt="image" src="https://github.com/isontheline/pro.webssh.net/assets/44212923/4d52ed44-c29f-4afa-8b65-0e7ad7530951">

Would you like to have another option to save the latest used?

**Comment by janwiesemann** [comment]
Somehow I've completely missed this option. I think it is adequate.

I think a icon or some kind of indication for the current mode would be great.

**Reply by isontheline** [reply]
@janwiesemann Thanks for your feedback, I will so enhance accordantly
